src/components/asset-sidebar/index.test.tsx

```diff
@@ -1,23 +1,25 @@
+import * as crypto from "crypto";
+
 import React from "react";
 
 import { render, server } from "@hackney/mtfh-test-utils";
 import { screen } from "@testing-library/react";
 import { rest } from "msw";
-import { v4 as uuidv4 } from "uuid";
 
 import { locale } from "../../services";
 
 import { AssetSideBar } from ".";
 
 import { Asset } from "@mtfh/common/lib/api/asset/v1";
+import { Patch } from "@mtfh/common/lib/api/patch/v1/types";
 import * as auth from "@mtfh/common/lib/auth/auth";
 
 const assetData: Asset = {
   id: "769894bd-b0bc-47eb-a780-322372c2448f",
   assetId: "0019062023",
   assetType: "Block",
-  patchId: uuidv4(),
-  areaId: uuidv4(),
+  patchId: crypto.randomBytes(20).toString("hex"),
+  areaId: crypto.randomBytes(20).toString("hex"),
```

Comment:
I think PR description should mention the npm package switch.
How come this was changed?